### PR TITLE
Add readme file for mrtcore sampleapps

### PR DESCRIPTION
**Pull request by rohanp-msft**

### CODE REVIEW

File: dev/MRTCore/TestApps/README.txt [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  <PropertyGroup Condition="'$(WindowsAppContainer)' != 'true'">
+    <AppxUploadPackageRecipe />
+    <AppxPackageRecipe />
+  </PropertyGroup>
+
+** Depending on whether the app is a C# or C++ one, the package will be in different locations.
+- In the case of C#, the app will use the package installed in the central location (e.g., C:\Users\rohanp.REDMOND\.nuget\packages).
+- In the case of C++, the package will be installed in the solution folder.
+
+Notes:
```
Comment (axelandrejs) [inline dev/MRTCore/TestApps/README.txt:22]: Remove  sausage making. Just state that this is a temporary requirement that will be addressed soon.

File: dev/MRTCore/TestApps/README.txt [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+- The reason this is needed:
+    - This script code imports the in-box APPX targets file for unpackaged app projects.
+    - When the Microsoft.ApplicationModel.Resources NuGet package is installed, the Microsoft.ApplicationModel.Resources.targets file will be imported, and we want that used instead of the in-box APPX targets file.
+- This will soon be changed in the WinUI 3.0 repo, after which this step will not be necessary (so long as you are using the new version of the NuGet package).
+
+4. Install the Microsoft.ApplicationModel.Resources NuGet package.
```
Comment (axelandrejs) [inline dev/MRTCore/TestApps/README.txt:28]: As we just discussed, this is moving soon. Let's leave this section blank and put a coming soon notice instead.

File: dev/MRTCore/TestApps/README.txt [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    a. Connect to the following feed: https://mscodehub.visualstudio.com/mrtcore/_packaging?_a=feed&feed=MRTCore.
+    b. Make sure you have the “Include prerelease” checkbox selected.
+Note: nuget.config in the root dir of the Reunion repo does not contain this feed as it is internal. So, you need to add it by yourself.
+
+Alternate, manual approach:
```
Comment (axelandrejs) [inline dev/MRTCore/TestApps/README.txt:33]: Remove the manual section.

File: dev/MRTCore/TestApps/README.txt [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+6. Now, use the Microsoft.ApplicationModel.Resources APIs! Sample code can be found at the bottom!
+- For the C# WinRT APIs, you need:
+    - using Microsoft.ApplicationModel.Resources;
+- For the C++/WinRT APIs, you need:
+    - #include "winrt/Microsoft.ApplicationModel.resources.h"
+    - using namespace winrt::Microsoft::ApplicationModel::Resources;
+- For the C APIs, you need:
+    - #include "MRM.h" 
+
+Microsoft.ApplicationModel.Resources API sample code
```
Comment (axelandrejs) [inline dev/MRTCore/TestApps/README.txt:49]: This is out of date and should be removed. The samples are now here.

**Review by axelandrejs — CHANGES_REQUESTED** [review]

File: dev/MRTCore/TestApps/README.txt (hunk 1 above)
Comment (huichen123) [reply dev/MRTCore/TestApps/README.txt:22]: has this not been addressed?

---
In reply to: [509507613](https://github.com/microsoft/ProjectReunion/pull/245#discussion_r509507613) [](ancestors = 509507613)

**Comment by huichen123** [comment]
readme should have extension md

File: dev/MRTCore/TestApps/README.txt (hunk 1 above)
Comment (rohanp-msft) [reply dev/MRTCore/TestApps/README.txt:22]: > readme should have extension md

Done.

Comment (rohanp-msft) [reply dev/MRTCore/TestApps/README.txt]: The latest versions of the WinUI NuGet package (not public yet) has our stuff included. That has this code removed. We likely cannot install our NuGet package alongside it lest there be conflicts.
The older versions of the WinUI NuGet package (which is what the sample apps use) does have this and needs to be removed.

File: dev/MRTCore/TestApps/README.txt (hunk 2 above)
Comment (rohanp-msft) [reply dev/MRTCore/TestApps/README.txt]: Please see my response above in the comment thread that starts with "Remove sausage making. Just state that this is a temporary requirement that will be addressed soon."

File: dev/MRTCore/TestApps/README.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    <AppxUploadPackageRecipe />
+    <AppxPackageRecipe />
+  </PropertyGroup>
+
+** Depending on whether the app is a C# or C++ one, the package will be in different locations.
+- In the case of C#, the app will use the package installed in the central location (e.g., C:\Users\rohanp.REDMOND\.nuget\packages).
+- In the case of C++, the package will be installed in the solution folder.
+
+2. Install the Microsoft.ApplicationModel.Resources NuGet package.
+    a. Connect to the following feed: https://mscodehub.visualstudio.com/mrtcore/_packaging?_a=feed&feed=MRTCore.
```
Comment (axelandrejs) [inline dev/MRTCore/TestApps/README.md:19]: Adding a note that this is a temporary location and will soon move is probably helpful.

**Review by axelandrejs — APPROVED** [review]
:shipit:

**Comment by huichen123** [comment]
It's full of internal stuff here. I am not sure whether we want to have this. 
If you do want this, this is md file. make sure it's formatted correctly.

**Comment by wjk** [comment]
My 2¢: A "coming soon" notice would do more harm than good here. I am getting quite sick of all the "coming soon" in this repo. No offense intended, but I want less talk and more code. This is the only code that has been released for Reunion so far, and I want to use it in my app today. Updating one line in my `nuget.config` file once the feed moves is a perfectly acceptable tradeoff.

If this is still unacceptable, _please_ provide a concrete timeline as to when these binaries will be publicly available without having to build from source. Progress on the Reunion proposals has been ridiculously slow, and is continuing to slow over time. I want to see this project continue to thrive. Thanks!

**Comment by axelandrejs** [comment]
Our apologies for the delay. We are working through some last issues to get this fully up-and-running. For now, the best way to try it is from source. We hope to fix this very soon.

**Pull request closed without merging**